views/partials/navbar.ejs

```diff
@@ -6,7 +6,7 @@
     <a href="/login">Login</a>
     <a href="/register">Register</a>
     <% } else { %>
-    <a href="/play">Play</a>
+    <a href="/play-settings">Play</a>
```

Comment:
How does it work, is there a check if there is a session in progress then redirects to /play to continue or you always go to /play-settings?